tests/resources/functions/dart-2.19/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: open_runtimes_test
+description: Dart runtime test
+version: 1.0.0
+publish_to: none
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
3.0 to 4.0, right?

Reply:
not for this one this is for 2.19 so it's good